azure/converters/futures.go

```diff
@@ -19,13 +19,13 @@ package converters
 import (
 	"encoding/base64"
 
-	azureautorest "github.com/Azure/go-autorest/autorest/azure"
+	autorestazure "github.com/Azure/go-autorest/autorest/azure"
```

Comment:
just curious why autorestazure vs. azureautorest (to me it seemed like the latter was more gramatically correct but I could be wrong)

Reply:
I'm not really sure why I chose that. I agree that `azureautorest` "reads" more naturally, and it has the advantage of being the most common alias already in use. I made that change.